go/iot/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"github.com/mgutz/logxi/v1"
+	"gobot.io/x/gobot"
+	"gobot.io/x/gobot/drivers/gpio"
+	"gobot.io/x/gobot/platforms/raspi"
+	"time"
+)
+
+// Copyright 2017, OpenCensus Authors
```

Comment:
Another nit: the license should be moved to the top line.

Reply:
Okay.